src/shared/components/input/Checkbox.tsx

```diff
@@ -166,17 +61,23 @@ export const Checkbox: FC<CheckboxProps> = ({
                 onBlur={() => {
                     setIsFocusWithin(false);
                 }}
+                className='h-0 w-0 opacity-0 z-[-1]'
             />
-            <Indicator
+            <div
```

Comment:
`div` cannot go inside `label`

Reply:
that breaks the use of the checkbox though 🤔 best to keep the label to make the text clickable as well instead of purely the checkbox. Also something actually broke the checkbox it seems as I can no longer select it (e.g. address settings > show passphrase)